ister.py

```diff
@@ -408,7 +413,7 @@ def copy_os(args, template, target_dir):
         swupd_env["http_proxy"] = template["HTTPProxy"]
         LOG.debug("http_proxy: {}".format(template["HTTPProxy"]))
 
-    run_command(swupd_command, environ=swupd_env)
```

Comment:
Is this a problem? Wouldn't the output be helpful regardless? I can easily fix with an arg to ister that is only used by the gui, however.